drivers/dma/dma_nxp_sdma.c

```diff
@@ -389,13 +391,15 @@ static int dma_nxp_sdma_reload(const struct device *dev, uint32_t channel, uint3
 
 	chan_data = &dev_data->chan[channel];
 
-	if (!size)
+	if (!size) {
```

Comment:
nit, as existing: as you're already changing the line, consider changing it to "size == 0" to make it clear that size isn't a boolean.